src/devices_models/devices/hydro_generation.jl

```diff
@@ -362,13 +362,13 @@ function budget_constraints!(psi_container::PSIContainer,
         device_budget_param_ub(psi_container,
                             budget_data,
                             Symbol("budget_$(H)"), # TODO: better name for this constraint
-                            UpdateRef{H}(:storage_capacity),
+                            UpdateRef{H}("get_storage_capacity"),
                             Symbol("P_$(H)"))
     else
-        device_budget_param_ub(psi_container,
-                            budget_data,
-                            Symbol("budget_$(H)"), # TODO: better name for this constraint
-                            Symbol("P_$(H)"))
+        device_budget_ub(psi_container,
+                        budget_data,
+                        Symbol("budget_$(H)"), # TODO: better name for this constraint
```

Comment:
Energy_limit?

Reply:
Sure, I can rename the constraint to Energy_limit